docs/sources/panels-visualizations/visualizations/xy-chart/index.md

```diff
@@ -0,0 +1,264 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/xy-chart/
+keywords:
+  - grafana
+  - chart
+  - xy chart
+  - documentation
+  - guide
+  - graph
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+description: Configure options for Grafana's xy chart
+title: XY chart
+weight: 100
+---
+
+# XY chart
+
+XY charts provide a way to visualize arbitrary x and y values in a graph so that you can easily show the relationship between two variables.
+
+![An xy chart showing height weight distribution](screenshot-xy-chart-v10.4.png)
+
+## Supported data formats
+
+You can use any type of tabular data in an xy chart. This type of visualization doesn't require time data.
+
+## XY chart options
+
+### Series mapping
+
+Set how series data is mapped in the chart.
+
+- [Auto](#auto-series-mapping-options) - Automatically generates series from all available data frames (or datasets). You can filter to select only one frame.
+- [Manual](#manual-series-mapping-options) - Explicitly define the series by selecting from available data frames.
+
+Depending on your series mapping selection, the **Frame**, **X-field**, and **Y-field(s)** options differ. The [Auto](#auto-series-mapping-options) and [Manual](#auto-series-mapping-options) series mapping sections describe those different settings.
+
+#### Auto series mapping options
+
+When you select **Auto** as your series mapping mode, the following options become available:
+
+- [Frame](#frame)
+- [X-field](#x-field)
+- [Y-fields](#y-fields)
+
+##### Frame
+
+By default, xy chart displays all data frames. You can filter to select only one frame.
```

Comment:
Not sure if XY is always capitalized, seems like it based on the page title?

Reply:
So for the page title, I've capitalized it because it would be awkward not to. But if it's not at the beginning of a sentence or in a title, then lower case it is.